### DISCUSSION TITLE
Ability to override Default Asset Location in the Field Layout Editor

### DISCUSSION THREAD

**Discussion by nitech** [craftcms/cms#8961] [Ideas]
### Description
My team is adding a lot of products to our Craft Commerce site. Because I am re-using the `productImages`-field accross all product types, my editors have to click their way through the asset hierarchy every time they add a new product.

I would like the ability to specify a `Default Asset Location` per implementation of the asset field (Field Layout Editor), in the same way as you can override the `Label` and `Instructions`.

![image](https://user-images.githubusercontent.com/1279436/99654313-6d0e2980-2a5a-11eb-9625-3c567d989872.png)

**Comment by brandonkelly** [comment]
Not currently possible to override certain field type-specific settings in the layout designer, but that has been requested (#4321), and once it is, the default asset location could be a candidate.

**Comment by nitech** [comment]
It would allow for a smoother content authoring experience.
